synapse/api/room_versions.py

```diff
@@ -102,12 +102,13 @@ class RoomVersions(object):
         enforce_key_validity=True,
         special_case_aliases_auth=True,
     )
-    MSC2260_DEV = RoomVersion(
-        "org.matrix.msc2260",
-        RoomDisposition.UNSTABLE,
+    MSC2432_DEV = RoomVersion(
```

Comment:
Now that I've realized this implements MSC2261 and MSC2432 this might not be the best name.

It is also possible that MSC2261 is just fully obsoleted by MSC2432.

Reply:
I think it's fine.